### DISCUSSION TITLE
Friday night funkin kade engine 1.6.1 how to character offsets

### DISCUSSION THREAD

**Discussion by Reyororo** [KadeArchive/Kade-Engine#1563] [Q&A] [answered]
I can't open debug mode by pressing keyboard 8. How can I open debug mode?

**Comment by isakube** [comment, marked as answer]
you open debug mode by running in debug mode
![Screen Shot 2021-08-03 at 7 00 28 AM](https://user-images.githubusercontent.com/68293280/128028699-75a2ee23-1e86-4f54-beba-ab8a08c5a119.png)
this but windows

edit: then go to your character offsets and copy what you find good there

**Reply by Reyororo** [reply]
Where should I go to run debug mode?

**Reply by CallMeInsane** [reply]
i believe you do lime test windows -debug
replace windows with ur operating system

**Reply by Reyororo** [reply]
solved! Thank you for your reply! I am grateful!
